spaces/S000190/properties/P000001.md

```diff
@@ -0,0 +1,16 @@
+---
+space: S000190
+property: P000001
+value: false
+refs:
+- doi: 10.1007/978-1-4612-6290-9
+  name: Counterexamples in Topology
+---
+
+$X$ is not $T_0$ because the only open set containing
+distinct points $a, b \in X$ is $X$ itself, and thus,
+there exists no open set containing $a$ but not $b$ or
+containing $b$ but not $a$.
```

Comment:
The explanation (cut and pasted from S4) does not make full sense as written (and did not make full sense for S4 either). The first part says that the only open set containing two distinct points in the whole set.  But this proves nothing for the $T_0$ property.  The second part is ok.  How about drastically simplifying all that and only give a hint of the reason (maybe the fact that the only nonempty open set is $X$ itself), or just referring to Counterexamples?

Reply:
I did one better - $T_0$ can be disproven by the indiscrete property so I just dropped the assertion and let the 🤖 do the work.